java/runtime/src/main/java/io/ray/runtime/exception/CrossLanguageException.java

```diff
@@ -0,0 +1,18 @@
+package io.ray.runtime.exception;
+
+import io.ray.runtime.generated.Common.Language;
+import io.ray.runtime.generated.Common.RayException;
+
+public class CrossLanguageException extends NativeRayException {
```

Comment:
I'm a little confused about the relationship between `CrossLanguageException` and `NativeRayException`. How do I know one should be used and the other should not? Can we merge them into one exception class? I see you've added only one new error class for Python, which is `CrossLanguageError`, so why we need two classes in Java?

Reply:
The cross language exception type is to make the stack more clear.